pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -195,12 +196,26 @@ type NetworkEndpointSettings struct {
 
 // ContainerFromNative instantiates a Docker-compatible Container from containerd-native Container.
 func ContainerFromNative(n *native.Container) (*Container, error) {
+	var mounts *specs.Spec
+	// for unmarshalling mount details
+	err := json.Unmarshal(n.Container.Spec.Value, &mounts)
```

Comment:
We already have `specs.Spec`. Please see the `if sp, ok := n.Spec.(*specs.Spec); ok {` line below.